internal/pkg/override/patch.go

```diff
@@ -114,6 +115,23 @@ func unmarshalPatches(path string, fs afero.Fs) ([]yamlPatch, error) {
 	return patches, nil
 }
 
+// addYAMLPatchDescription updates the Description field of a CloudFormation
+// to indicate it has been overriden with YAML patches for us to keep track of usage metrics.
+func addYAMLPatchDescription(body *yaml.Node) {
+	if body.Kind != yaml.DocumentNode {
+		return
+	}
+	body = body.Content[0] // Move inside the document.
+	for i := 0; i < len(body.Content); i += 2 {
```

Comment:
i'm fine to leave it this way too
```suggestion
    i, err := findInMap(body, "Description", nil)
	if err != nil {
    }
```
https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/override/patch.go#L259